charms/precise/clearwater-bono/hooks/programmable-multiple-relation-changed

```diff
@@ -2,16 +2,11 @@
 set -e
 
 # Set our DNS requirements
-TTL=300
 id=$(cut -d/ -f2 <<< $JUJU_UNIT_NAME)
 ip=$(unit-get public-address)
 relation-set domain=$(config-get zone)
-relation-set resources='@ '$TTL' IN A '$ip'
-@ '$TTL' IN NAPTR 1 1 "S" "SIP+D2T" "" _sip._tcp
-@ '$TTL' IN NAPTR 2 1 "S" "SIP+D2U" "" _sip._udp
-_sip._tcp '$TTL' IN SRV 0 0 5060 bono-'$id'
-_sip._udp '$TTL' IN SRV 0 0 5060 bono-'$id'
-bono-'$id' '$TTL' IN A '$ip
+DNS_JSON=`$CHARM_DIR/lib/generate_dns_records $ip @ bono-$id 5060 _sip._tcp _sip._udp`
```

Comment:
For consistency (and future-proofness, and readability), should probably use `$(...)` rather than backticks.